docs/getting-started/ea-req.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[ea-requirements]]
+= Entity Analytics prerequisites
+
```

Comment:
> @SourinPaul are we now calling the risk scoring feature "Entity Analytics" in the docs? Also, should this page be included under the Useful links section within the Entity Risk Score page? Linking to multiple doc pages in the UI is a little excessive imo, but I definitely think feature requirements are necessary to highlight.


@nastasha-solomon the current UI reference to the feature is incorrect. Here is the bug logged to correct the UI reference #7920. In its correct form, the UI reference will take the user to the (new) `Entity Risk Scoring` feature documentation page.

I agree that linking to multiple doc pages (Feature overview, Prerequisites for the feature) in the UI may be excessive, given we expect to improve or evolve the feature quickly. 

How about ensuring the `Entity Risk Scoring Prerequisites` section is highlighted on the page the user first lands ( section above)?

Reply:
Thanks for sharing that issue, @SourinPaul. Good to know that the links are being updated for 8.11! :)

As for linking this page, I do think it should be placed at the top of the list. I'd use the page title as well, so the linked text is consistent with our docs. 

Lastly, it might be a good idea to reference this page wherever these privileges are needed to view risk score data within the Security app. For example, if these privileges are required to turn on and or view risk score in the Entity Analytics dashboard, I would mention it there. I'm not sure what that'd look like, or if there's even time to add it into 8.11, so just something to consider for a future release.